### PR TITLE
Added Swatch change logic for React example

### PR DESCRIPTION
**Pull request by dxArtemiusz**
Updated theme context
Added swatch themes
Updated SCSS variables
Added CSS variables
Changed appearance of Theme Selector

### CODE REVIEW

File: react/src/components/theme-selector/ThemeSelector.js
Comment (16adianay) [inline react/src/components/theme-selector/ThemeSelector.js:1]: You can create a separate object for SelectBox|dropDownOptions

Comment (16adianay) [inline react/src/components/theme-selector/ThemeSelector.js:1]: And for the TextBox|inputAttr:)

File: react/src/contexts/theme.js
Comment (16adianay) [inline react/src/contexts/theme.js:1]: You can try using the useEvent hook instead of useCallback with functions with dependecies

Comment (artem-kurchenko) [reply react/src/contexts/theme.js:1]: applySwatchTheme  and applySwatchTheme should be either defined outside or memorized

**Review by artem-kurchenko — APPROVED** [review]

**Pull request merged**